lib/puppetserver/ca/action/migrate.rb

```diff
@@ -66,6 +66,10 @@ def run(input)
         def migrate(old_cadir, new_cadir=PUPPETSERVER_CA_DIR)
           FileUtils.mv(old_cadir, new_cadir)
           FileUtils.symlink(new_cadir, old_cadir)
+          # Ensure the symlink has the same ownership and permissions as the actual cadir
+          cadir = File.stat(new_cadir)
+          File.chown(cadir.uid, cadir.gid, old_cadir)
+          File.chmod(cadir.mode, old_cadir)
```

Comment:
`chmod` doesn't actually work on symlinks. From the man page:

> chmod never changes the permissions of symbolic links; the chmod system call cannot change their permissions. This is not a problem since the permissions of symbolic links are never used.  However, for each symbolic link listed on the command line, chmod changes the permissions of the pointed-to file.  In contrast, chmod ignores symbolic links encountered during recursive directory traversals.

Reply:
So we're thinking we can just leave this out then? Makes sense to me.

Reply:
yeah, I'm gonna remove this.